### PR TITLE
Add collectibles embed modal and deep linking for collectibles

### PR DESCRIPTION
**Pull request by Kyle-Shanks**
### Description

Client updates for collectibles embed modals and deep linking to collectibles

Currently the deep linking works, but it causes a refresh when the route changes on the ProfilePage. Will attempt to fix this 

Also the 'Copied to Clipboard' toast  shows up at the top of the page for the embed button in the CollectiblesPage dropdown menu. This requires an update of the Toast component to fix so this is a fine solution for now.

### How Has This Been Tested?

Lots of manual testing and review from the team

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/kj-Add-embed-modal-for-collectibles

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/kj-Add-embed-modal-for-collectibles

File: src/containers/collectibles/components/CollectiblesPage.tsx [hunk 1]
```diff
@@ -394,15 +506,22 @@ const CollectiblesPage: React.FC<{
...
-              type={ButtonType.COMMON}
-              size={ButtonSize.TINY}
-              text={collectibleMessages.edit}
-              leftIcon={<IconPencil />}
-              onClick={handleEditClick}
-            />
-          )}
+          <PopupMenu
+            items={overflowMenuItems}
+            // TODO: Bottom left is offset a bit. Need to look into that
```
Comment (raymondjacobson) [inline src/containers/collectibles/components/CollectiblesPage.tsx:511]: what should this be doing?

File: src/containers/collectibles/components/CollectiblesPage.tsx [hunk 2]
```diff
@@ -534,6 +655,58 @@ const CollectiblesPage: React.FC<{
...
+        bodyClassName={styles.embedModalBody}
+        titleClassName={styles.embedModalTitle}
+        headerContainerClassName={styles.embedModalHeader}
+        allowScroll
+        zIndex={zIndex.COLLECTIBLE_EMBED_VIEW_MODAL}
+      >
+        <div className={styles.embedContainer}>
+          <div
+            className={styles.embedFrame}
+            dangerouslySetInnerHTML={{ __html: embedIFrameUrl }}
```
Comment (raymondjacobson) [inline src/containers/collectibles/components/CollectiblesPage.tsx:673]: can we not reuse this component? https://github.com/AudiusProject/audius-client/blob/be456a65c14b2a2c6e85b734c2b258f0f122eb37/src/containers/embed-modal/components/EmbedFrame.tsx#L16

**Review by raymondjacobson — APPROVED** [review]

File: src/containers/collectibles/components/CollectibleDetails.tsx [hunk 3]
```diff
@@ -60,9 +60,9 @@ const CollectibleDetails: React.FC<{
   }, [mediaType, frameUrl, gifUrl, name, setFrame, setIsLoading])
 
   const handleItemClick = useCallback(() => {
-    dispatch(setCollectible({ collectible }))
-    setIsModalOpen(true)
-  }, [collectible, dispatch, setIsModalOpen])
+    // @ts-ignore
+    navigate(`/${match.params.handle}/collectibles/${getHash(collectible.id)}`)
```
[the remark this answers is not in the document]
Comment (Kyle-Shanks) [reply src/containers/collectibles/components/CollectibleDetails.tsx:64]: ts doesn't think that params has a handle property when it does

File: src/assets/img/iconEmbed.svg [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<svg width="18" height="18" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg">
```
[the remark this answers is not in the document]
Comment (Kyle-Shanks) [reply src/assets/img/iconEmbed.svg:1]: 💯

File: src/containers/collectibles/components/CollectibleDetails.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (raymondjacobson) [reply src/containers/collectibles/components/CollectibleDetails.tsx:64]: gotcha. maybe worth a comment

File: src/containers/collectibles/components/CollectiblesPage.tsx (hunk 1 above)
Comment (Kyle-Shanks) [reply src/containers/collectibles/components/CollectiblesPage.tsx:511]: When I set it to bottom left to match the designs, it looks like this so this was a note to look into the component to try to fix in stems
Screen Shot 2021-11-16 at 2 40 40 PM

Comment (raymondjacobson) [reply src/containers/collectibles/components/CollectiblesPage.tsx:511]: gotcha 👍

File: src/containers/collectibles/components/CollectiblesPage.tsx (hunk 2 above)
Comment (Kyle-Shanks) [reply src/containers/collectibles/components/CollectiblesPage.tsx]: good call, not sure why I didn't the first time

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/kj-Add-embed-modal-for-collectibles

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/kj-Add-embed-modal-for-collectibles

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/kj-Add-embed-modal-for-collectibles

**Review by sddioulde — APPROVED** [review]
looks good!

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/kj-Add-embed-modal-for-collectibles

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/kj-Add-embed-modal-for-collectibles

**Pull request merged**